testsuite/tests/mgc/conftest.py

```diff
@@ -103,13 +112,18 @@ def gateway(upstream_gateway, spokes, hub_policies_commit):
     return gw
 
 
+@pytest.fixture(scope="module")
+def openshift(gateway):
+    """OpenShift client for the primary namespace"""
+    return gateway.openshift
```

Comment:
I think this would cause troubles with circular fixtures in the feature tests.

Reply:
Could you name which tests you see as potential issues? I ran all the tests (at least with gateway_api.enabled) and it all passed.